packages/e/element/package.yml

```diff
@@ -55,25 +68,34 @@ build      : |
     yarn run build:native
     yarn run build
 install    : |
-    install -dm00644 $installdir/usr/bin
-    install -dm00644 $installdir/usr/share/element
-    cp -Rv dist/linux-unpacked/* $installdir/usr/share/element
-    cp -Rv element-web/webapp $installdir/usr/share/element/resources/
-    ln -sv ../share/element/element-desktop $installdir/usr/bin/element-desktop
+    export eddir=/usr/share/element
```

Comment:
Moving this to `environment:|` probably makes the packaging more readable.

Also, I might have missed something, but why not name it `$element_installdir` and prefix it with `$installdir`?